### DISCUSSION TITLE
I Broke the HomeBridge Ring Plug-in

### DISCUSSION THREAD

**Discussion by EddieDaYankee** [dgreif/ring#901] [Q&A]
Are you able to help me with this?

I Broke the HomeBridge Ring Plug-in

I had the HomeBridge Ring plug-in working on my Mac. However, I could not hear any sound from my cameras. According to the logs, there was some sort of ffmpeg(?) issue. I followed the link and tried to troubleshoot the problem myself. Based on what I read, I entered the following into Terminal on my Mac:

sudo curl -Lf# https://github.com/homebridge/ffmpeg-for-homebridge/releases/latest/download/ffmpeg-darwin-x86_64.tar.gz | sudo tar xzfm

Now, I can’t get live streams to work. I’m not sure what I did. I uninstalled and reinstalled the plugin but it did not work. My Mac is currently running macOS 10.13. 

Are you able and willing to help me fix this? Thank you in advance.

**Comment by bjurusik** [comment]
Any update on this?  Same issue happened to me - everything had been working (including sound).  Ended up doing a clean MacOS install and re-installed Homebridge, and now I get the error that ffmpeg was not found no matter how many times I try to install it.

**Comment by EddieDaYankee** [comment]
Hi,
I just posted the following on Reddit in response to someone else with a similar problem. Maybe it can help you too. Before my Ring stopped showing live feeds on HomeKit (without sound), I was getting a similar FFMPEG error. Since I can’t seem to get this fixed on my Mac I came up with the following:

That was my next move, but I tried installing it (Scrypted) on my Mac the other day and wasn’t successful. Strangely, there is very little documentation out there regarding how to install and run Scrypted. From what I could gather, I needed to install HomeBrew which I did. It fell apart after that. However, it occurred to me today that there is an iOS app called “Hub” which lets you run HomeBridge on a spare iDevice. I’m hoping that running the Ring Plug-In on a dedicated spare iPhone will solve my problem. Not sure if this will help either of us, but it’s more help than anyone offered me. Good luck 🍀!

Maybe this will give you something to go on.  Let me know if you have any success. Thanks.
